packages/graphback-codegen-resolvers/src/output/createResolvers.ts

```diff
@@ -53,53 +53,49 @@ export function generateCustomCRUDResolvers(schema: GraphQLSchema, models: Model
     return outputResolvers;
 }
 
-export function createMutations(graphqlType: GraphQLObjectType, crudOptions: GraphbackCRUDGeneratorConfig) {
-    const objectName = graphqlType.name.toLowerCase();
-
+export function createMutations(modelName: string, crudOptions: GraphbackCRUDGeneratorConfig) {
     const mutations = {};
     if (crudOptions.disableGen) {
         return mutations;
     }
 
     if (crudOptions.create) {
-        const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.CREATE);
+        const fieldName = getFieldName(modelName, GraphbackOperationType.CREATE);
         // tslint:disable-next-line: no-any
-        mutations[fieldName] = createTemplate(objectName, crudOptions.subCreate)
+        mutations[fieldName] = createTemplate(modelName, crudOptions.subCreate)
     }
     if (crudOptions.update) {
-        const fieldName = getFieldName(graphqlType.name, GraphbackOperationType.UPDATE);
-        mutations[fieldName] = updateTemplate(objectName, crudOptions.update);
+        const fieldName = getFieldName(modelName, GraphbackOperationType.UPDATE);
+        mutations[fieldName] = updateTemplate(modelName, crudOptions.update);
```

Comment:
I think we need to first do https://github.com/aerogear/graphback/issues/668 to visualize how this will work, as passing string to CRUD layer might be confusing. 

WE can have:
- New crud layer for each Type that accepts field mapping and id in constructor.
- Generic CRUD layer that will accepts string as Model name but then Service layer and crud layer need to map it to the table. 
- Pass the table name 

Some discussion needs to happen there